homeassistant/components/deconz/binary_sensor.py

```diff
@@ -14,6 +30,16 @@
 ATTR_TILTANGLE = "tiltangle"
 ATTR_VIBRATIONSTRENGTH = "vibrationstrength"
 
+DEVICE_CLASS = {
+    CarbonMonoxide: DEVICE_CLASS_GAS,
+    Fire: DEVICE_CLASS_SMOKE,
+    GenericFlag: "",
```

Comment:
A missing device class should return `None`. We can leave the generic flag out and look up the dict with `dict.get`.

Reply:
I agree